staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -295,6 +300,18 @@ func ListResource(r rest.Lister, rw rest.Watcher, scope RequestScope, forceWatch
 		ctx := scope.ContextFunc(req)
 		ctx = request.WithNamespace(ctx, namespace)
 
+		exportOptions := metav1.ExportOptions{}
+		if err := metainternalversion.ParameterCodec.DecodeParameters(req.URL.Query(), scope.MetaGroupVersion, &exportOptions); err != nil {
+			err = errors.NewBadRequest(err.Error())
+			scope.err(err, w, req)
+			return
+		}
+		if exportOptions.Export && e == nil {
+			err = errors.NewBadRequest(fmt.Sprintf("export of %q is not supported", scope.Resource.Resource))
```

Comment:
you don't need the exporter here, right? if export=true routed to List, export is not supported